app/views/stories/_form_errors.html.erb

```diff
@@ -24,7 +24,7 @@
         <%= text_area_tag "comment", @comment&.comment, :rows => 5 %>
         <br><br>
       </div>
-    <% elsif story.similar_stories.any? %>
+    <% elsif story.similar_stories.any? && story.is_saved_by_cur_user %>
```

Comment:
Why call this method?

Reply:
That CR was fast 😅 
was my first attempt trying to solve the issue before actually  went to the partial and notice the flag.

Reply:
@pushcx 
btw, not totally sure the images represent well enough the scenario for the current issue... any feedback is appreciated.